plugins/net_plugin/net_plugin.cpp

```diff
@@ -4039,6 +4015,29 @@ namespace eosio {
       return fc::json::from_string(s).as<T>();
    }
 
+   size_t net_plugin_impl::parse_connection_limit( const std::string& limit_str) {
```

Comment:
Maybe rename to `parse_connection_rate_limit`

Reply:
Renamed.